afs/afs-core/src/main/resources/lang/Project.properties

```diff
@@ -0,0 +1 @@
+Project=Project
```

Comment:
English as default language is a good idea, but we probably have to remove the Project_en.properties to avoid full duplication.

Another idea is to create a utility function to do the job